client/service/src/client/wasm_overwrite.rs

```diff
@@ -15,6 +15,27 @@
 
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! # WASM Local Blob-Overwrite
+//!
+//! WASM Local blob overwrite provides tools to replace on-chain WASM with locally-built WASM.
+//! These locally-built WASM blobs may include functionality that is not included in the
```

Comment:
```suggestion
//! These overwritten WASM blobs may include functionality that is not included in the
```

"overwritten", but I'm not happy with this either xD

Reply:
"customized"?

Reply:
Sounds good :)